security/client-certificate-validator/src/main/resources/META-INF/services/fish.payara.security.api.ClientCertificateValidator

```diff
@@ -0,0 +1 @@
+fish.payara.example.security.certificate.MyValidator
```

Comment:
This file needs to be renamed to `fish.payara.security.client.ClientCertificateValidator` because the package changed.

Reply:
Hi @rdebusscher, please also see this. This file needs to be renamed too to match the new package name.